plugins/hydra_ax_sweeper/example/conf/config.yaml

```diff
@@ -19,14 +19,14 @@ hydra:
       # Number of epochs without a significant improvement from
       # the currently known best parameters
       # An Epoch is defined as a batch of trials executed in parallel
-      max_epochs_without_improvement: 10
+      max_epochs_without_improvement: 3
     
     params:
       banana.x:
         type: range
-        bounds: [-5, 10.0]
+        bounds: [-10, 10]
         log_scale: false
       banana.y:
         type: choice
-        values: [-1, 0, 1, 3, 4, 5]
+        values: [-3, -2, -1, 0, 1, 2, 3, 4]
```

Comment:
Why are you making y a choice? for banana function it makes no sense.

Reply:
I wanted to add an example of using "Choice parameter"

Reply:
I suggest either in README or in a second function where it makes sense.

Reply:
README has this. Reverting to Range